CONTRIBUTING.md

```diff
@@ -1,119 +1,66 @@
-
-Ensure you have an Amazon AWS account. Once you do, login and go to the IAM (Identity & Access Management) service. There, you'll do the following things:
-
-1. If you prefer, create a user for this service.
-2. With the user you want to use for this purpose, get an Access Key and Secret Key from IAM. Make sure to record these in a convenient place (a password keeper, for example), as the Secret Key cannot be retrieved once initially provided.
-3. Create an IAM Policy named `Product Advertising API` as follows:
-  ```{json}
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Action": "ProductAdvertisingAPI:*",
-            "Resource": "*"
-        }
-    ]
-}
-```
-4. Attach the IAM Policy you created to the user you created for whom you the keys.
-
-Once these steps are complete, add the keys to your `.env` file as follows:
-
-```{env}
-AWS_ACCESS_KEY_ID=<Access Key here>
-AWS_SECRET_ACCESS_KEY=<Secret Key here>
-```
-
-#### Setting up local environment via Docker
+1. Go to the green "Clone or download" button on right of screen of the repository frame and copy that URL. It will be of the format `https://github.com/<YOUR OWN GITHUB REPOSITORY>/florence-api.git` (this URL will be different if you are using SSH)
+2. In the directory on your local box in which you plan to work: `git clone https://github.com/<YOUR OWN GITHUB REPOSITORY>/florence-api.git`
+3. Press `enter` and watch git fetch the repository for you into a sub-directory named `florence-api`
 
-##### Docker
+### Option 1: Setting up local environment via Docker
 
 1. Follow the instructions that can be found within the Docker site starting [here](https://www.docker.com/get-started).
-2. Once you have Docker installed and running execute the following command to build and deploy your containers locally by executing the following command from the project root directory:
-```sh
-RAILS_ENV=development docker-compose up --build
-```
-3. When you are done development work, don't forget to bring everything down gracefully by executing the following command from the project root directory:
-```sh
-docker-compose down
-```
+2. Once you have Docker installed and running execute the following command to build and deploy your containers locally by executing the following command from the project root directory: `RAILS_ENV=development docker-compose up --build`
+3. Edit your files, work on data, develop a feature; do whatever you intend to do, at this point.  When you are finished, commit, push to github, and create a pull request if you intend to share the results of your effort with the rest of the community.
+4. When you are done development work, don't forget to bring everything down gracefully by executing the following command from the project root directory:
+`docker-compose down`
 
-#### Setting up local environment manually
+### Option 2: Manually setting up local environment
```

Comment:
Do we want to have a links to installing Ruby/PostgreSQL in a local environment here?

Reply:
Yes.  Do the following resources look good?

- Postgres: https://wiki.postgresql.org/wiki/Detailed_installation_guides
- Ruby: https://www.ruby-lang.org/en/documentation/installation/
- Rails: https://guides.rubyonrails.org/getting_started.html

Reply:
sorry, yes.